berth_reservations/schema.py

```diff
@@ -1,17 +1,17 @@
+import applications.schema
```

Comment:
`applications` should be added to `setup.cfg`'s `[isort]` as `known_first_party`

Reply:
and now once you've done that, isort will fail 🙂